src/tab/tabs.ts

```diff
@@ -77,18 +77,28 @@ export class RemixTabs extends LitElement {
     if (index !== -1) {
       this.tabs = [...this.tabs.slice(0, index), ...this.tabs.slice(index + 1, this.tabs.length)];
     }
-    // send message to the parent to remove one tab and update the property
+    this.dispatchEvent(id)
+  }
+
+  /** activate a specific tab from the list */
+  public activateTab({ detail: id }: CustomEvent) {
+    const index = this.tabs.findIndex(tab => tab.id === id)
+    if (index !== -1) {
+      this.activated = id;
+    }
+    this.dispatchEvent(id)
```

Comment:
No you cannot do that. You need to dispatch a CustomEvent, not an id. You should do : 
```typescript
public activateTab(event: CustomEvent) {
  const index = this.tabs.findIndex(tab =>tab.id === event.detail)
  ...
  this.dispatchEvent(even)
}
```